src/libraries/skinDynLib/test/skinContactTest.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2017 Fondazione Istituto Italiano di Tecnologia (IIT)
+ * CopyPolicy: Released under the terms of the GNU GPL v2.0.
+ */
+
+#include <cassert>
+#include <ctime>
```

Comment:
Correct me if I am wrong, but I was unable to find an equivalent to `std::clock` in `chrono`.